modules/notification/mail/mail.go

```diff
@@ -104,7 +104,12 @@ func (m *mailNotifier) NotifyIssueChangeAssignee(doer *models.User, issue *model
 
 func (m *mailNotifier) NotifyPullReviewRequest(doer *models.User, issue *models.Issue, reviewer *models.User, isRequest bool, comment *models.Comment) {
 	if isRequest && doer.ID != reviewer.ID && reviewer.EmailNotifications() == models.EmailNotificationsEnabled {
-		ct := fmt.Sprintf("Requested to review #%d.", issue.Index)
+		var ct string
+		if issue.Repo.UnitEnabled(models.UnitTypeExternalTracker) {
+			ct = fmt.Sprintf("Requested to review !%d.", issue.Index)
```

Comment:
```suggestion
			ct := fmt.Sprintf("Requested to review %s.", issue.HTMLURL())
```
This should end up with the correct result since issue.HTMLURL() will have the full URL to the pull and then when it is rendered it will be converted to the expected ```#2``` type format with the link pointing at the correct place

Reply:
cunning plan